creusot-contracts/src/std/clone.rs

```diff
@@ -5,6 +5,10 @@ extern_spec! {
     mod std {
         mod clone {
             trait Clone {
+                // TODO:
+                // Requiring result == *self seems too strong since objects can
+                // contain information which is lost by the clone (e.g., capacity of
+                // vectors...).
```

Comment:
Is that the case?

Reply:
It's really not clear to me.

If we want things like the capacity of vectors to be available in the specs, then the spec of Clone breaks. If we want the addresses of Boxes, same thing (this might be needed for interior mutability).

Reply:
What do we do then? Compare the models?

Reply:
Seems like a reasonable compromise.